lib/fake_idp/configuration.rb

```diff
@@ -59,7 +59,7 @@ def default_algorithm
     end
 
     def default_encryption
-      ENV["ENCRYPTION_ENABLED"] || true
+      !%w[FALSE OFF F NO].include?(ENV["ENCRYPTION_ENABLED"].to_s.upcase) && true
```

Comment:
@ireneybean First, thanks for contributing!

I think there may be an opportunity to simplify this and make encryption purely opt-in and instead default to false. Also, I think if this environment variable is set to `""` or `nil` (implicitly disabled) encryption would always be enabled which, as I think about this a bit more, might be unexpected for some users.

@cbortz What do you think?

Reply:
Yep, turning encryption off with this patch would require specifically setting it to "false". It looked to me like y'all were trying have to have it default to true so I was trying to respect that, but allow it to be disabled via the environment variable. It doesn't make too much difference to me personally which is the default, as long as I can turn it off.  If having it default to off works for you, I'm good with that. I'll update this to work that way.